frontend/src/app/pages/exercises/exercise/shared/exercise-map/utility/feature-manager.ts

```diff
@@ -26,6 +26,11 @@ export interface FeatureManager<ElementFeature extends Feature<any>> {
         feature: ElementFeature
     ) => void;
 
+    /**
+     * @returns whether the feature can be translated
+     */
```

Comment:
This comment is actually not giving anymore information as is already provided by the name. I would propose to maybe write into the comment what translatable means for a feature (or link to a documentation explaining it) or just remove it all together (but I would propose either explain or to put a link in)?

Reply:
```suggestion
    /**
     * @returns whether the feature can be moved by the user
     */
```